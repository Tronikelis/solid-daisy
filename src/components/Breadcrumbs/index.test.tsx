import { render } from "@solidjs/testing-library";
import { describe, expect, it } from "vitest";

import { Breadcrumbs, BreadcrumbsItem } from ".";

describe("<Breadcrumbs />", () => {
    it("renders correctly", () => {
        const { container } = render(() => (
            <Breadcrumbs>
                <BreadcrumbsItem>Foobar</BreadcrumbsItem>
                <BreadcrumbsItem>BarFoo</BreadcrumbsItem>
            </Breadcrumbs>
        ));

        expect(container).toMatchSnapshot();
    });
});
