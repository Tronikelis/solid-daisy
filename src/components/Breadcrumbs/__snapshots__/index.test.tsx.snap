// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Breadcrumbs /> > renders correctly 1`] = `
<div>
  <div
    class="breadcrumbs text-sm"
  >
    <ul>
      <li>
        Foobar
      </li>
      <li>
        BarFoo
      </li>
    </ul>
  </div>
</div>
`;
